backends/rapidpro/status.go

```diff
@@ -103,7 +103,7 @@ UPDATE msgs_msg SET
 		WHEN 
 			:status = 'E' 
 		THEN 
-			NOW() + (5 * (error_count+1) * interval '1 minutes') 
+			GREATEST(NOW() + (5 * (error_count+1) * interval '1 minutes'), :next_attempt::timestamp with time zone)
```

Comment:
I'm not sure `:next_attempt::timestamp with time zone` is valid syntax - `:foo` syntax is just a way for the sqlx library to created queries that bind to struct fields but they ultimately get replaced with regular `$1`, `$2` etc param names.

Reply:
I changed to use a string for interval that we can handle well easily in the SQL

Reply:
#411